decidim-elections/app/controllers/decidim/elections/elections_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    class ElectionsController < ApplicationController
+      include Decidim::ApplicationHelper
+      include Decidim::AttachmentsHelper
+      include Decidim::FilterResource
+      include Decidim::Elections::Orderable
+      include Decidim::Paginable
+
+      helper_method :elections, :election, :tab_panel_items, :questions, :paginated_elections
+
+      def index; end
+
+      def show
+        raise ActionController::RoutingError, "Not Found" unless election
+
+        respond_to do |format|
```

Comment:
```suggestion
        respond_to do |format|
```

Reply:
This is using the same approach as the proposals_controller, you think we should not send the "not found"?